fairlearn/metrics/_disaggregated_result.py

```diff
@@ -105,7 +108,7 @@ def apply_grouping(
         self,
         grouping_function: str,
         control_feature_names: Optional[List[str]] = None,
-        errors: str = "raise",
+        errors: ErrorHandler = "raise",
```

Comment:
```suggestion
        errors: "raise" | "coerce" = "raise",
```

Reply:
Hey @adrinjalali , did you mean `errors: Literal["raise, "coerce"] = "raise"` ? I don't think  `errors: "raise" | "coerce" = "raise"` is a valid syntax, is it ? :thinking:

Reply:
It's valid python syntax, but mypy and pylance complain. So I'm okay with using `Literal` here. pity.